### PR TITLE
ICTV Import

### PR DESCRIPTION
**Pull request by spiekos**
Note: need to add Notes and Caveats as well as commands for running and testing scripts. Also need to add tests and testing files

### CODE REVIEW

File: scripts/biomedical/ICTV_Taxonomy/scripts/tests.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#!/bin/bash
+
+java -jar /Applications/datacommons-import-tool-0.1-alpha.1-jar-with-dependencies.jar lint tMCFs/virusMasterSpeciesList.tmcf CSVs/VirusSpecies.csv ICTV*.mcf
+mv dc_generated species
+
+java -jar /Applications/datacommons-import-tool-0.1-alpha.1-jar-with-dependencies.jar lint tMCFs/virusTaxonomy.tmcf CSVs/VirusIsolates.csv ICTV*.mcf
+mv dc_generated virus_isolates
+
+java -jar /Applications/datacommons-import-tool-0.1-alpha.1-jar-with-dependencies.jar lint tMCFs/virusGenomeSegment.tmcf CSVs/VirusGenomeSegments.csv ICTV*.mcf
+mv dc_generated genome_segments
```
Comment (pradh) [inline scripts/biomedical/ICTV_Taxonomy/scripts/tests.sh:31]: I think this script is very specific to where you downloaded the file to.

Maybe as the first step, you could download `https://github.com/datacommonsorg/import/releases/download/0.1-alpha.1k/datacommons-import-tool-0.1-alpha.1-jar-with-dependencies.jar` to `/tmp` and then run it?

It also assumes the user has `java` JRE installed, which you can point out in the comments maybe.

File: scripts/biomedical/ICTV_Taxonomy/scripts/create_virus_taxonomic_ranking_enums.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+"""
+
+# load environment
+import pandas as pd
+import sys
+
+
+# declare universal variables
+HEADER = [
+'sort',\
```
Comment (pradh) [inline scripts/biomedical/ICTV_Taxonomy/scripts/create_virus_taxonomic_ranking_enums.py:31]: nit: I don't think you need `\` for lists etc that are delimited by `,`

File: scripts/biomedical/ICTV_Taxonomy/README.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
+
+# Importing Master Species List and Virus Metadata Resource from the International Committee on Taxonomy of Viruses (ICTV)
```
Comment (pradh) [inline scripts/biomedical/ICTV_Taxonomy/README.md:2]: Could you please check the test failures in https://github.com/datacommonsorg/data/pull/834/checks?check_run_id=22280933775

I think you'll need to fix lint (`./run_tests.sh -f`)

**Review by pradh — COMMENTED** [review]
Thanks for the detailed doc!

I only skimmed through the tests, one main comment on the validation script.

File: scripts/biomedical/ICTV_Taxonomy/scripts/create_virus_taxonomic_ranking_enums.py (hunk 2 above)
Comment (spiekos) [reply scripts/biomedical/ICTV_Taxonomy/scripts/create_virus_taxonomic_ranking_enums.py]: removed from all three python script files

File: scripts/biomedical/ICTV_Taxonomy/scripts/tests.sh (hunk 1 above)
Comment (spiekos) [reply scripts/biomedical/ICTV_Taxonomy/scripts/tests.sh:37]: updated the tests script so downloading the tool is part of the script and the comments in the README.md to explain this functionality in the tests subsection

**Review by pradh — APPROVED** [review]
Thanks for the updates!

File: scripts/biomedical/ICTV_Taxonomy/README.md (hunk 3 above)
Comment (spiekos) [reply scripts/biomedical/ICTV_Taxonomy/README.md:2]: ran the lint tests!

**Pull request merged**